docs/style-guide.rst

```diff
@@ -650,6 +650,50 @@ No::
         doSomething();
     }
 
+Multiline output parameters and return statements should follow the same style recommended for `wrapping long lines <http://solidity.readthedocs.io/en/develop/style-guide.html#maximum-line-length>`_.
```

Comment:
You should use internal links here. I think they have a format like `:ref:maximum-line-length`, but you have to check.

Reply:
done ✓